src/Common/utils.ts

```diff
@@ -1,5 +1,38 @@
+import { Utils } from 'xpring-common-js/build/src'
+
 const isNode = (): boolean => {
   return process?.release?.name === 'node'
 }
 
+/**
+ * Converts a string such that each of its characters are represented as hex.
+ *
+ * @param value - the string to convert to hex.
+ * @returns the hex encoded version of the string.
+ */
+export const stringToHex = (value: string): string => {
+  let result = ''
+  // eslint-disable-next-line no-plusplus
```

Comment:
@hbergren To explain if this is correct or not?

Reply:
I suspect there's an easy way to use `Buffer` to do this without the ned for a loop?

Reply:
Yup, we should be able to do something like:

```ts
Buffer.from(value).toString('hex')
```

But if it's that easy, I wonder if we need the `stringToHex` helper at all.

Reply:
I had followed an example on stackoverflow that was probably out of date. I can remove the helper if you prefer.